demos/Ollama Chat Service/ollama.html

```diff
@@ -0,0 +1,26 @@
+<html>
+<head>
+  <title>Experimenting with Ollama in a Browser</title>
+</head>
+<body>
+  <script type="module">
+    import "./ollama.js";
+  </script>
+  <h1>Experimenting with Ollama in a Browser</h1>
```

Comment:
Thanks for adjusting the demo titles in README. Can you also adjust <h1> and <title> texts in actual demos? When running any of these demos, it's helpful to know which version it is.

Reply:
Yes, that makes sense.  And, I think the <h1> should match the new <title> as well.  I guess that is what you were pointing out, and not the headings in the `README.md`.

Okay, fixed.